sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -3551,6 +3787,53 @@ public partial class PerfCounterDataSource : System.ClientModel.Primitives.IJson
         public static bool operator !=(Azure.ResourceManager.Monitor.Models.PerfCounterDataSourceStream left, Azure.ResourceManager.Monitor.Models.PerfCounterDataSourceStream right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class Pipeline : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Pipeline>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Pipeline>
```

Comment:
this name is quite generic, could we add some prefix to specify it?
I also saw that there is a resource added in this PR, if we change this name, maybe the resource name needs to change accordingly.

Reply:
Renamed to `PipelineGroupServicePipeline`.